src/common/types/chainner-scope.ts

```diff
@@ -64,10 +64,23 @@ struct NcnnNetwork {
 
 struct OnnxFile { path: string }
 struct OnnxModel {
-    scale: int(1..),
-    inputChannels: int(1..),
-    outputChannels: int(1..),
+    arch: string,
+    subType: string,
+    scaleHeight: int(1..) | null,
+    scaleWidth: int(1..) | null,
 }
+let OnnxRemBgModel = OnnxModel {
+    arch: "u2net",
```

Comment:
Is arch even useful anymore?

Reply:
Maybe not? I was just copying the way PyTorch does this.

Reply:
This may possibly be used in the future if we ever get ONNX broadcasting fully. I'm not sure.

Reply:
Eh, I can just remove it for now and add it back if it's needed.